### PR TITLE
Ldc patch

### PR DESCRIPTION
**Pull request by hover2pi**
Minor fixes to make the LDC work. Plus updated the notebook.

Also verified that the mu values are being scaled correctly (bug pointed out by Jeff) and the grism results are yielding different coefficients for each bin (bug pointed out by Giovanni).

### CODE REVIEW

File: ExoCTK/notebooks/limb_darkening_demo.ipynb [hunk 1]
```diff
@@ -121,14 +119,34 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "12/1056 spectra in parameter range Teff: (4500, 4600) , logg: (5, 5.5) , FeH: (-0.5, 0.5) , wavelength: (1.1, 1.9)\n",
+      "12/12 spectra in parameter range Teff:  (2500, 2600) , logg:  (5, 5.5) , FeH:  (-0.5, 0.5) , wavelength:  (0, 40)\n",
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply ExoCTK/notebooks/limb_darkening_demo.ipynb]: It actually doesn't matter. If you trim the wavelength, the calculation is slightly faster but it's not necessary. The flux is multiplied by the grism throughput and everything outside is cut off anyway.

File: ExoCTK/modelgrid.py [hunk 2]
```diff
@@ -210,8 +211,43 @@ def __init__(self, model_directory, bibcode='2013A & A...553A...6H',
...
+        
+        # Update the wavelength
+        wave = model['wave']
+        hdu[0].header['CRVAL1'] = min(wave)
+        hdu[0].header['CDELT1'] = np.mean(np.diff(wave))
+        hdu[0].header['CUNIT1'] = 'Micron'
+        
+        # Write the file
+        hdu.writeto(filepath)
+
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply ExoCTK/modelgrid.py]: Gotcha.

File: ExoCTK/notebooks/limb_darkening_demo.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (julesfowler) [reply ExoCTK/notebooks/limb_darkening_demo.ipynb]: Ahhh, gotcha. A note to this effect might be useful? But I trust your judgement.

**Comment by julesfowler** [comment]
@hover2pi Check out the slack convo I tagged you in -- this is some sort of Travis CI issue, and Marshall and Joe came up with the solution.

**Comment by julesfowler** [comment]
@hover2pi Are you ready for a review on this? 
Do you know why the CI is failing?

**Comment by hover2pi** [comment]
Merge #110 instead.

**Pull request closed without merging**